api/src/main/java/org/apache/gravitino/policy/Policy.java

```diff
@@ -93,6 +95,15 @@ public interface Policy extends Auditable {
    */
   Content content();
 
+  /**
+   * Get the list of job templates that are associated with the policy. The job templates can be
+   * used to execute jobs that are related to the policy.
+   *
+   * @return The list of job templates that are associated with the policy, can be empty if no job
+   *     templates are associated with the policy.
+   */
+  List<JobTemplate> associatedJobTemplates();
```

Comment:
Is it possible there will be multiple job template for one policy?

Reply:
I will keep this possibility, as there's potentially a scenario that one policy needs several templates to coordinate.